src/seedu/addressbook/data/person/Address.java

```diff
@@ -54,8 +54,10 @@ public static boolean isValidAddress(String test) {
 
     @Override
     public String toString() {
-        return block.getBlock() + "," + ;
-    }
+        return block.getBlock() + ", " 
+        	   + street.getStreet()  + ", " 
+        	   + unit.getUnit() + ", " 
+        	   +postal_code.getPostalCode();
```

Comment:
- Remember to be consistent and have a space after the plus symbol.
- You could reuse ADDRESS_SPLITER here to be consistent.
- Note: Concatenating multiple strings in Java using this format is well-known to be inefficient. While this line of code could be made more efficient, it should not matter that much since you're only dealing with a small number of concatenations here.